### PR TITLE
chore: refresh PR template

### PR DESCRIPTION
**Pull request by Chriscbr**
Updates our pull request template with a helpful checklist for contributors.

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: docs/06-contributors/030-pull-requests.md [hunk 1]
```diff
@@ -54,15 +54,15 @@ changelog. To that end, pull request titles must follow this convention:
   * `repo` - change related to repository behavior (e.g. pull request templates, issue templates,
     etc).
 * `<subject>`
-  * Should be all lowercase and without a period (improves aesthetics of changelog).
+  * Prefer lowercase (except for proper nouns) and do not include a period (improves aesthetics of changelog).
```
Comment (ShaiBer) [inline docs/06-contributors/030-pull-requests.md:57]: Wouldn't the linter fail us on any upper case?

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
 
+Unless this is a very simple 1-line-of-code change, please create a new issue describing the change you're proposing first, then link to it from this PR.
+
+Read more about our process in our contributing guide: https://docs.winglang.io/contributors/pull_requests
+
+-->
+
+### Related issue
+
+Fixes # <!-- INSERT ISSUE NUMBER -->
```
Comment (ShaiBer) [inline .github/pull_request_template.md:11]: maybe add an example for multiple fixes?

**Review by ShaiBer — APPROVED** [review]

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+### Checklist
+
+- [ ] I have updated the PR title to match [Winglang's style guide](https://docs.winglang.io/contributors/pull_requests#how-are-pull-request-titles-formatted)
+- [ ] I have performed a self-review of my code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation if applicable
+- [ ] I have added tests that prove my fix is effective or that my feature works if applicable
+- [ ] My code changes how tests are executed or modifies code that might be platform-sensitive (related to paths, etc.), so I would like the label `pr/e2e-full` to be added to run tests on all platforms
+
+<!-- If this is still a work in progress, feel free to open a draft PR until you're able to check off all the items on the list above -->
```
Comment (hasanaburayyan) [inline .github/pull_request_template.md:26]: Should we add a misc section?

Comment (Chriscbr) [reply .github/pull_request_template.md]: I removed a lot of the extra text per some feedback

**Pull request merged**